go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	cosmossdk.io/x/feegrant v0.1.1
 	cosmossdk.io/x/tx v0.13.7
 	cosmossdk.io/x/upgrade v0.1.2
+	github.com/CosmWasm/wasmd v0.51.0
```

Comment:
should be using wasmd v54 -- this only requires `cosmos/cosmos-sdk v0.50.11` which manifest can bump too as well. does need `wasmvm/v2 v2.2.1` too

Reply:
This bumps `ibc` to `8.4.0`. Will this cause any problems?

Reply:
Fixed in 52fc64ba99a9f452d1d0bd955cda9b6f186629d0